### PR TITLE
[Components with more than 1 service] LPS-192612 Refactor ModelListener components part II

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-192612

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:analytics-cloud-acceptance

**Comment by MrJohn1911** [comment]
ci:test:analytics-cloud

**Comment by catarineaguiar** [comment]
no regressions found

**Review by rafaprax — APPROVED** [review]

**Review by dantewang — CHANGES_REQUESTED** [review]
@MrJohn1911 `EntityModelListener` is renamed to `AnalyticsEntityModel`, but there are some methods, like `BaseModelListener.getEntityModelListener` that are not updated. Please update, thanks.

**Comment by MrJohn1911** [comment]
In the last push, I renamed a method in this [commit](https://github.com/liferay-core-infra/liferay-portal/pull/5531/commits/3fe63ff2045cda8c56e130f2b6f321eb92911fca), from `getEntityModelListener` to `getAnalyticsEntityModel`

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
no regressions found

File: modules/apps/analytics/analytics-message-sender-impl/src/main/java/com/liferay/analytics/message/sender/internal/util/AnalyticsModelUtil.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+				attributeNames.add(name);
+			}
+		}
+
+		return attributeNames;
+	}
+
+	public static boolean isCustomField(
+		UnsafeFunction<Long, ExpandoTable, Exception>
```
Comment (shuyangzhou) [inline modules/apps/analytics/analytics-message-sender-impl/src/main/java/com/liferay/analytics/message/sender/internal/util/AnalyticsModelUtil.java:53]: @MrJohn1911 this is too weird. Just change the user to use expandoTableLocalService.fetchExpandoTable() and pass in the ExpandoTable instance. And inside this `isCustomField()` method you check for null.

**Comment by shuyangzhou** [comment]
@MrJohn1911 See above comment, please resend.

**Pull request closed without merging**